Week3/More Javascript/script.js

```diff
@@ -0,0 +1,148 @@
+//1
+
+function info(a, b, c) {
+    console.log(a + b + c)
+}
+info(1, 5, 34);
+
+//2
+
+function colorCar(z) {
```

Comment:
Nice! But the variable name `z` doesn't really explain what the variable will do. Do you know a better name for that variable?

Reply:
Got it. :)